service/sidecar/mapping.go

```diff
@@ -295,12 +295,15 @@ func checkMetaNamespace(txNs *protoblocktx.TxNamespace) protoblocktx.Status {
 	}
 
 	nsUpdate := make(map[string]any)
-	u := policy.GetUpdatesFromNamespace(txNs)
+	u, err := policy.GetUpdatesFromNamespace(txNs)
+	if err != nil {
+		return protoblocktx.Status_MALFORMED_NAMESPACE_POLICY_INVALID
+	}
 	if u == nil {
 		return statusNotYetValidated
 	}
 	for _, pd := range u.NamespacePolicies.Policies {
-		_, err := policy.ParseNamespacePolicyItem(pd)
+		_, err := policy.CreateNamespaceVerifier(pd, nil)
```

Comment:
minor: If I understand correctly, the deserializer is not needed because it is only used when evaluating signatures. Since this policy is created only to validate its form, we don't need the deserializer.
If this is true, please add a comment for that.

Reply:
Added.